### ISSUE TITLE
menu always centered without <center> tag

### ISSUE THREAD

**Issue by hanwiz** [foundation/foundation-emails#653]
**How can we reproduce this bug?**

Write out the HTML (or Inky code) that causes the issue.

```
            <wrapper>
                <h5>Summary text goes here.</h5>                
                <menu class="vertical">
                    <item href="#">&bull; First Section</item>
                    <item href="#">&bull; Second Section</item>
                    <item href="#">&bull; Just a Plain Link</item>
                    <item href="#">&bull; Just a Plain Link</item>
                    <item href="#">&bull; Just a Plain Link</item>
                </menu>                
            </wrapper>
```

**What did you expect to happen?**

Each menu item should be left aligned.

**What happened instead?**
Each item is centered with 'float-center' class

**What email clients does this happen in?**
Every browser/client.

**Comment by dmhendricks** [comment]
I agree that it seems odd to automatically add `float-center` to the menu items. I think a better default would be to left-aligned as well.

**Comment by stevenMouret** [comment]
+1

**Comment by davehoran** [comment]
Has anyone else been able to reproduce this?

**Comment by stevenMouret** [comment]
I think this happens when a center tag is positioned before the menu. In my case the center is just after the body. Therefore not a direct parent.

**Comment by ghost** [comment]
Was this fixed or what is the solution to align it left? I got still a problem aligning it left in Outlook clients

**Comment by LibertusArator** [comment]
+1

**Comment by LibertusArator** [comment]
node_modules Inky componentFactory.js `// <center>` remove `element.find('item, .menu-item').addClass('float-center');`...

**Comment by Aerendir** [comment]
I have the same problem... Is there a way to override this default behaviour?

How can I force to left-align the items?

**Comment by milianriedel** [comment]
+1

**Comment by joeworkman** [comment]
Foundation for Emails has been superseded by [Inky v2](https://github.com/foundation/inky), a complete rewrite in Rust with improved Outlook support, hybrid output mode, bulletproof buttons, validation, and more. This issue will not be addressed in the legacy codebase.

If this issue persists in v2, please open a new issue at https://github.com/foundation/inky/issues
